content/blog/recipes-0-1-15/index.Rmd

```diff
@@ -0,0 +1,135 @@
+```
+
+Speaking of missing data, we've decided to rename the current eight imputation steps:
+
+* `step_impute_knn()` is favored over `step_knnimpute()`
+* `step_impute_median()` is favored over `step_medianimpute()`
+* and so on...
+
+These are a lot better since they work well with tab-complete. The old steps will go through a gradual deprecation process before being removed at some point in the future. 
+
+## Keeping columns used in other features
+
+A fair number of steps take one or more columns of the data and convert them to artificial features. For example, principal component regression represents a set of columns as artificial features that are amalgamations of the original data. In some cases, users desired to be able to keep the original columns. 
+
+The following steps have an option called `keep_original_cols`: `step_date()`, `step_dummy()`, `step_holiday()`, `step_ica()`, `step_isomap()`, `step_kpca_poly()`, `step_kpca_rbf()`, `step_nnmf()`, `step_pca()`, `step_pls()`, and `step_ratio()`. 
+
+For example: 
+
+```{r keep-cols}
+scat_rec <- 
+  recipe(Species ~ Location + d13C + d15N + CN, data = scat) %>% 
+  step_impute_mean(d13C, d15N, CN) %>% 
+  step_dummy(all_nominal_predictors(), one_hot = TRUE) %>% 
+  step_pca(d13C, d15N, CN, keep_original_cols = TRUE) %>% 
+  prep()
+
+scat_rec %>% 
+  bake(new_data = scat) %>% 
+  names()
+```
+
+## Acknowledgements
+
+Thakns for everyone who contributed since the previous version: [&#x0040;AshesITR](https://github.com/AshesITR), [&#x0040;BenoitLondon](https://github.com/BenoitLondon), [&#x0040;CelloJuan](https://github.com/CelloJuan), [&#x0040;dfalbel](https://github.com/dfalbel), [&#x0040;EmilHvitfeldt](https://github.com/EmilHvitfeldt), [&#x0040;gregdenay](https://github.com/gregdenay), [&#x0040;gustavomodelli](https://github.com/gustavomodelli), [&#x0040;hfrick](https://github.com/hfrick), [&#x0040;hsbadr](https://github.com/hsbadr), [&#x0040;jake-mason](https://github.com/jake-mason), [&#x0040;jjcurtin](https://github.com/jjcurtin), [&#x0040;juliasilge](https://github.com/juliasilge), [&#x0040;konradsemsch](https://github.com/konradsemsch), [&#x0040;kylegilde](https://github.com/kylegilde), [&#x0040;LePeti](https://github.com/LePeti), [&#x0040;LordRudolf](https://github.com/LordRudolf), [&#x0040;lukasal](https://github.com/lukasal), [&#x0040;mattwarkentin](https://github.com/mattwarkentin), [&#x0040;mikemc](https://github.com/mikemc), [&#x0040;mine-cetinkaya-rundel](https://github.com/mine-cetinkaya-rundel), [&#x0040;paudel-arjun](https://github.com/paudel-arjun), [&#x0040;renanxcortes](https://github.com/renanxcortes), [&#x0040;rorynolan](https://github.com/rorynolan), [&#x0040;saadaslam](https://github.com/saadaslam), [&#x0040;schoonees](https://github.com/schoonees), [&#x0040;topepo](https://github.com/topepo), [&#x0040;uriahf](https://github.com/uriahf), [&#x0040;vadimus202](https://github.com/vadimus202), and [&#x0040;zenggyu](https://github.com/zenggyu).
```

Comment:
```suggestion
Thanks to everyone who contributed since the previous version: [&#x0040;AshesITR](https://github.com/AshesITR), [&#x0040;BenoitLondon](https://github.com/BenoitLondon), [&#x0040;CelloJuan](https://github.com/CelloJuan), [&#x0040;dfalbel](https://github.com/dfalbel), [&#x0040;EmilHvitfeldt](https://github.com/EmilHvitfeldt), [&#x0040;gregdenay](https://github.com/gregdenay), [&#x0040;gustavomodelli](https://github.com/gustavomodelli), [&#x0040;hfrick](https://github.com/hfrick), [&#x0040;hsbadr](https://github.com/hsbadr), [&#x0040;jake-mason](https://github.com/jake-mason), [&#x0040;jjcurtin](https://github.com/jjcurtin), [&#x0040;juliasilge](https://github.com/juliasilge), [&#x0040;konradsemsch](https://github.com/konradsemsch), [&#x0040;kylegilde](https://github.com/kylegilde), [&#x0040;LePeti](https://github.com/LePeti), [&#x0040;LordRudolf](https://github.com/LordRudolf), [&#x0040;lukasal](https://github.com/lukasal), [&#x0040;mattwarkentin](https://github.com/mattwarkentin), [&#x0040;mikemc](https://github.com/mikemc), [&#x0040;mine-cetinkaya-rundel](https://github.com/mine-cetinkaya-rundel), [&#x0040;paudel-arjun](https://github.com/paudel-arjun), [&#x0040;renanxcortes](https://github.com/renanxcortes), [&#x0040;rorynolan](https://github.com/rorynolan), [&#x0040;saadaslam](https://github.com/saadaslam), [&#x0040;schoonees](https://github.com/schoonees), [&#x0040;topepo](https://github.com/topepo), [&#x0040;uriahf](https://github.com/uriahf), [&#x0040;vadimus202](https://github.com/vadimus202), and [&#x0040;zenggyu](https://github.com/zenggyu).
```